strings/ion_laws.json

```diff
@@ -87,9 +87,6 @@
         "INVISIBLE",
         "PAINFUL",
         "HARMFUL",
-        "HOMOSEXUAL",
-        "HETEROSEXUAL",
-        "SEXUAL",
```

Comment:
Correct, however from seeing in action the resulting chat communications that happen is what this'd resolve.